api/full_api.json

```diff
@@ -1396,7 +1396,10 @@
             "$ref": "#/components/parameters/csf"
           },
           {
-            "$ref": "#/components/parameters/limit"
+            "$ref": "#/components/parameters/pageSize"
```

Comment:
Did you change the URI parameter name from `limit` to `pageSize`?
I think the NGSI-LD spec says `limit` (personally I like `pageSize` a lot more - almost proposed it yesterday :))